components/playlist/playlist_download_request_manager.cc

```diff
@@ -175,6 +173,10 @@ void PlaylistDownloadRequestManager::OnGetMedia(base::Value value) {
     LOG(ERROR) << value;
```

Comment:
not technically part of this PR but shouldn't this be VLOG?

also if this is happening in the main process, seems `value` should be passed through the JSON sanitizer before calling  `base::JSONWriter::Write` to comply with https://chromium.googlesource.com/chromium/src/+/master/docs/security/rule-of-2.md